tslint.json

```diff
@@ -1,4 +1,5 @@
 {
+    "extends": "tslint:latest",
```

Comment:
If this is, as you say in the PR description, going to cause a large number of tslint warnings then let's leave it out for now and enable important warnings individually until we are in a position to do a blanket enable.  Or we could do a full tslint immediately, but I'd keep that as a separate PR.  We should avoid having a lot of errors in the build, and I don't want to overwhelm the CI changes by having a barrage of lint-related changes in the same PR.  Does that make sense?

Reply:
Agree, we can disable the `latest` rules first. And enable it on separate PR later.